includes/course-video/class-sensei-course-video-settings.php

```diff
@@ -97,6 +130,10 @@ public function enqueue_frontend_scripts() {
 		$script              = "window.sensei = window.sensei || {}; window.sensei.courseVideoSettings = $video_settings_json;";
 
 		wp_add_inline_script( 'sensei-course-video-blocks-extension', $script, 'before' );
+
+		if ( has_block( 'core/video', $post ) || has_block( 'core/embed', $post ) ) {
```

Comment:
The previous approach would check if it's a VideoPress, YouTube, or Vimeo embed. We'll include the script if the page has any embed, regardless the block variation. But I don't think it's a big deal since it's not very big assets, right?